### DISCUSSION TITLE
Package name a deprecated item comes from

### DISCUSSION THREAD

**Discussion by abigailbeets-wk** [dominikh/go-tools#961] [Q&A] [answered]
Hi, when analyzing for deprecated things, is it possible to also get the name of the package that it originates from? For example, if I use a deprecated function from httputil, can I get the output to tell me that I'm using a deprecated function from that package specifically, rather than just the piece of my code that is using it?

**Comment by dominikh** [comment, marked as answer]
There is not, no. The assumption is that the user goes to the location indicated in the output (which points at the specific identifier), and then uses their usual code navigation tools.

**Reply by abigailbeets-wk** [reply]
Thank you! I was curious about how this tool might be used in CI.

**Reply by dominikh** [reply]
We could make the output more verbose. Right now, given the following call to the deprecated method `Meth`

```
func fn(x bar.T) {
	x.Meth()
}
```

we print `x.Meth is deprecated: <the deprecation message>`. We could extend this to `x.Meth ((*github.com/some/package.Type).Meth) is deprecated: <the deprecation message>` but I worry that this might be too verbose.

**Comment by tuyenkec1** [comment]
I need a copy of the victim's cookie eating software on chrome coco operamini, across browsers can you help me
